requirements/verification-by-reference.md

```diff
@@ -24,7 +24,7 @@ This doc experiments with the avoidance of signatures as the reference.
 1. A collection of verification objects may be associated with an artifact.
 1. Leverage the garbage collection infrastructure registry operators have implemented.
    Garbage collection represents significant investments for registry services which are based on OCI Manifest and OCI Index.
-   We should aim to utilize these existing schemas, or only slight verifications from them to maximize the opportunity for registry implementations to adopt Notary v2.
+   We should aim to utilize these existing schemas, or only slight verifications from them to maximize the opportunity for registry implementations to adopt Notary Project specification implementation.
```

Comment:
> Notary Project specification implementation

What is the specification implementation? Are you looking for registries to adopt the specification in their implementation? Or, are you looking for them to adopt Notation? I'm not entirely sure from the grammar there.

Reply:
I will update it to `implementations of the Notary Project specification`. There could be different implementations based on the Notary Project specification in `notaryproject` repo. cc @toddysm for comments.

Reply:
it will be updated to `Notary Project signature specification` with a link to the spec.